runtime/crab/src/migration.rs

```diff
@@ -45,47 +45,5 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> frame_support::weights::Weight {
-	// dawinia
-	use darwinia_staking::CacheState;
-	if let Some(s) = migration::get_storage_value::<(CacheState, CacheState, CacheState)>(
-		b"DarwinaStaking",
```

Comment:
It seems that here is also a typo, does it mean this migration was not successfully executed?

What's the impact for Crab.

Reply:
Get `None`, then skip the `put`.

Fallback to use the https://github.com/darwinia-network/darwinia/blob/1ac5283f5baaf351d472de87bc162d373ee4897e/pallet/staking/src/lib.rs#L244. This only affects the first session. The election might also be skipped.

Reply:
If `ExposureCacheStates ` is renamed and not used anymore, should we clean its storage?

Reply:
There is no need for now. There are many unnecessary items. Clean them all up in Staking V3 at once.